snap/snapcraft.yaml

```diff
@@ -1,5 +1,5 @@
 name: multipass
-version: '2017.0.0'
+version: '2018.0.0'
```

Comment:
If you're touching it, do `git` instead, there's no point having any one version here.

Reply:
I'm not sure how that would be accomplished. Could you please point to an example on how to do that?

Reply:
Ah, ok, sure, I can do that:)

Reply:
Ok, fixed.